package.json

```diff
@@ -118,7 +118,6 @@
   },
   "dependencies": {
     "@babel/core": "^7.11.1",
```

Comment:
Are we still using babel in production? I thought we pre-babel'd everything during the build?

Reply:
We still babel-register, but I plan to remove that soon, once the plugin-helpers take care of babel-ifying things at build time.